tools/test-static-code

```diff
@@ -55,4 +55,19 @@ else
     echo "ok 3 js-translatable-strings"
 fi
 
+#
+# Unsafe content-security-policy
+#
+# It's dangerous to have 'unsafe-inline' or 'unsafe-eval' in our
+# content-security-policy entries. This is the browser equivalent
+# of setenforce 0
+#
+
+if grep -E 'content-security-policy.*(\*|unsafe)' pkg/*/*.json*; then
```

Comment:
This new test now needs to bump the total count on the top. I fixed that during rebasing, pushing now.